src/more-infos/more-info-configurator.html

```diff
@@ -140,14 +144,14 @@
       fields: this.fieldInput,
     };
 
-    this.isConfiguring = true;
+    this.attributes.is_configuring = true;
```

Comment:
What is `this` here? If you are trying to mutate the state object: don't. Those should be treated as immutable. You can have a temporary override stored as a property instead.